src/test/java/com/visenze/visearch/ViSearchTest.java

```diff
@@ -211,4 +211,11 @@ public void testMatchSearch() {
         visearch.matchSearch(searchParams);
         verify(searchOperations).matchSearch(searchParams);
     }
+
```

Comment:
@davidhuang01 can add in the Response parser test? to verify the `tags` response is parsed correctly into the map? Thank you.

Reply:
Sure, I'll add it soon.

Reply:
OK, I've added an additional test for recommendation response parsing in ViSearchSearchOperationsTest .